dll/win32/setupapi/CMakeLists.txt

```diff
@@ -44,6 +44,6 @@ add_library(setupapi MODULE
 set_module_type(setupapi win32dll UNICODE)
 target_link_libraries(setupapi uuid wine ${PSEH_LIB} oldnames)
 add_delay_importlibs(setupapi comdlg32 shell32 winspool.drv wintrust)
-add_importlibs(setupapi gdi32 comctl32 advapi32 user32 rpcrt4 version msvcrt kernel32 ntdll)
+add_importlibs(setupapi gdi32 comctl32 advapi32 user32 rpcrt4 version msvcrt kernel32 ntdll shell32)
```

Comment:
importlibs order is important, I don't think shell32 should be after ntdll.

Reply:
Ok, thanks! Is it a rule to be followed? E.g. the dependecies should be on the right?

Reply:
I don't remember exactly, but iirc this affects the final binaries, I've sent code suggestion.

Reply:
Why do we need to add shell 32 while it's already in the delay loads?